### ISSUE TITLE
Mechanics don't or incorrectly interact with shulker boxes

### ISSUE THREAD

**Issue by ApeironTsuka** [EngineHub/CraftBook#1184]
Pipe pistons break shulker boxes (although 1 transfer gets through beforehand), and MC1202 (Container Dispenser), at the very least, doesn't pull from shulker boxes. I didn't test other ICs.

**Comment by Mcknife96** [comment]
Yeah this is really an issue, but for the time being a workaround would look like this:

![image](https://user-images.githubusercontent.com/42012558/122426358-7b8cd700-cf90-11eb-8f02-bfe3f3b30f89.png)

Even though the observer clock and the pison are not connected, the observer fools the plugin into thinking they are somehow.
